weather-api-gateway/src/main/java/org/tenkichannel/weather/api/gateway/resources/WeatherResource.java

```diff
@@ -1,19 +1,14 @@
 package org.tenkichannel.weather.api.gateway.resources;
 
-import javax.inject.Inject;
-import javax.ws.rs.Consumes;
-import javax.ws.rs.GET;
-import javax.ws.rs.POST;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
-import javax.ws.rs.Produces;
-import javax.ws.rs.core.MediaType;
-
-import org.apache.camel.quarkus.core.runtime.CamelRuntime;
-import org.tenkichannel.weather.api.gateway.routes.WeatherGatewayRoute;
+import org.apache.camel.CamelContext;
 import org.tenkichannel.weather.api.gateway.domain.Forecast;
 import org.tenkichannel.weather.api.gateway.domain.Location;
 import org.tenkichannel.weather.api.gateway.domain.Weather;
+import org.tenkichannel.weather.api.gateway.routes.WeatherGatewayRoute;
+
+import javax.inject.Inject;
+import javax.ws.rs.*;
```

Comment:
IMHO use * on imports is not a good practice.

Reply:
Dammit IntelliJ. Do you know how do I fix that?

Reply:
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html